.gitignore

```diff
@@ -6,3 +6,7 @@ pubspec.lock
 test_integration/*.json
 test_integration/client_id*.yaml
 test_integration/config.yaml
+
+# IDE files
```

Comment:
please, no!

These should be in your own global ignore file

Reply:
Removed and thanks, many years using git and didn't know you could have a global ignore file!